cmd/argo/commands/lint.go

```diff
@@ -22,6 +22,7 @@ func NewLintCommand() *cobra.Command {
 		Use:   "lint FILE...",
 		Short: "validate files or directories of workflow manifests",
 		Run: func(cmd *cobra.Command, args []string) {
+			var unvalidWf error
```

Comment:
Be specific that this is an error in the variable name
```suggestion
			var invalidWfErr error
```